src/main/java/com/bloomtech/asylumrgbea/controllers/AsylumCaseController.java

```diff
@@ -20,8 +20,10 @@ public class AsylumCaseController {
     /**
      * As the point of entry this method is mapped to the default end point and provides all ResponseDtos.
      * @return An Iterable of AsylumCaseResponseDto with totalPage or an ErrorDto with a message.
+     *
+     * This end-point has been deprecated at this point in development. Removed @GetMapping annotation
```

Comment:
A very big YES to ample, clear comments.

Reply:
I love that we can leave deprecated annotation